app/assets/javascripts/misc/pw-strength.js

```diff
@@ -46,6 +54,9 @@ function analyzePw() {
   const pwCntnr = document.getElementById('pw-strength-cntnr');
   const pwStrength = document.getElementById('pw-strength-txt');
   const pwFeedback = document.getElementById('pw-strength-feedback');
+  const submit = document.querySelector('input[type="submit"]');
+
+  disableSubmit(submit, 0);
```

Comment:
Just to avoid  duplicating code. We need to disable the button in the keyup handler as well, so it seemed cleaner to call the function here, rather than writing another one off statement that did the same thing.